### ISSUE TITLE
训练自己的数据集

### ISSUE THREAD

**Issue by jianghe9** [Duankaiwen/CenterNet#16]
如何训练自己的数据集进行目标检测？数据集如何准备？谢谢。

**Comment by Duankaiwen** [comment]
You can format your own dataset in the same format as coco.

**Comment by HRanWang** [comment]
@Duankaiwen Can we load the pretrained CenterNet-104  model params to train our own data,  then how to set the learning rate? Thx.

**Comment by Duankaiwen** [comment]
@HRanWang Hi, I think you can use the default learning rate at the beginning，if the loss were very unstable, you could reduce the learning rate.

**Comment by HRanWang** [comment]
@Duankaiwen Thanks! Then how to change the input_size (511 , 511) and output_sizes (128 , 128), how to design it?

**Comment by Duankaiwen** [comment]
@HRanWang modify the “input_size” and "out_sizes" in the caonfig/xxx.json. Ceiling(input_size/out_sizes)=4

**Comment by zychen2016** [comment]
你训练完了吗？花了多长时间多少卡？

**Comment by mukekke** [comment]
@HRanWang Can you tell me how to use the pretrained model to retrain my own data?
